libraries/AP_HAL_ChibiOS/hwdef/PixC4-Jetson/hwdef.dat

```diff
@@ -0,0 +1,326 @@
+define HAL_BATT2_VOLT_PIN 2
+define HAL_BATT2_CURR_PIN 3
+
+# CAN bus
+PI9  CAN1_RX CAN1
+PH13 CAN1_TX CAN1
+
+PB12 CAN2_RX CAN2
+PB13 CAN2_TX CAN2
+
+PH2 GPIO_CAN1_SILENT OUTPUT PUSHPULL SPEED_LOW LOW GPIO(70)
+PH3 GPIO_CAN2_SILENT OUTPUT PUSHPULL SPEED_LOW LOW GPIO(71)
+PH4 GPIO_CAN3_SILENT OUTPUT PUSHPULL SPEED_LOW LOW GPIO(72)
+
+# GPIOs
+PA7 HEATER_EN OUTPUT LOW GPIO(80)
+define HAL_HEATER_GPIO_PIN 80
+
+PG1 VDD_BRICK_VALID INPUT PULLUP
+PG2 VDD_BRICK2_VALID INPUT PULLUP
+PG3 nVBUS INPUT PULLDOWN
+PA9 VBUS INPUT
+PF13 VDD_5V_HIPOWER_OC INPUT PULLUP
+PE15 VDD_5V_PERIPH_OC INPUT PULLUP
+
+# capture pins
+PA5 FMU_CAP1 INPUT  GPIO(58)
+PB3 FMU_CAP2 INPUT  GPIO(59)
+PB11 FMU_CAP3 INPUT GPIO(60)
+PI0 FMU_SPARE_4 INPUT GPIO(61)
+
+# SPI devices
+# note that SPI6 is disabled by default, try to enable it once we get one sensor up
+SPIDEV ms5611	       SPI4 DEVID5  BARO_CS    MODE3  5*MHZ  5*MHZ
+SPIDEV icm20602        SPI1 DEVID1  ICM20602_CS  MODE3  2*MHZ  8*MHZ
+SPIDEV icm20602_2      SPI1 DEVID2  ICM20602_CS2  MODE3  2*MHZ  8*MHZ
+SPIDEV icm42688        SPI1 DEVID3  ICM42688_CS  MODE3  2*MHZ  8*MHZ
+SPIDEV ramtron         SPI2 DEVID1  FRAM_CS      MODE3  8*MHZ  8*MHZ 
+
+# 3 IMUs
+IMU Invensense SPI:icm20602 ROTATION_YAW_90
+IMU Invensense SPI:icm20602_2 ROTATION_YAW_90
+IMU Invensensev3 SPI:icm42688 ROTATION_PITCH_180
+
+define HAL_DEFAULT_INS_FAST_SAMPLE 1
+
+# probe external I2C compasses plus some internal IST8310
+# we also probe some external IST8310 with a non-standard orientation
+define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+COMPASS RM3100 I2C:0:0x20 false ROTATION_NONE
+
+# baro
+BARO MS56XX      SPI:ms5611
+
+# microSD support
+PC8 SDMMC1_D0 SDMMC1
+PC9 SDMMC1_D1 SDMMC1
+PC10 SDMMC1_D2 SDMMC1
+PC11 SDMMC1_D3 SDMMC1
+PC12 SDMMC1_CK SDMMC1
+PD2 SDMMC1_CMD SDMMC1
+
+# red LED marked as B/E
+PB1 LED_RED OUTPUT OPENDRAIN GPIO(90)
+
+# green LED of FMU RGB LED
+PC6 LED_GREEN OUTPUT GPIO(91) LOW
+
+# blue LED of FMU RGB LED
+PC7 LED_BLUE OUTPUT GPIO(92) HIGH
+
+# setup for Board LEDs
+define HAL_GPIO_A_LED_PIN 90
+define HAL_GPIO_B_LED_PIN 92
+define HAL_GPIO_LED_ON 0
+
+# enable RAMTROM parameter storage
+define HAL_STORAGE_SIZE 16384
+define HAL_WITH_RAMTRON 1
+
+# allow to have have a dedicated safety switch pin
+define HAL_HAVE_SAFETY_SWITCH 1
+
+#define HAL_COMPASS_DEFAULT HAL_COMPASS_NONE
+define HAL_COMPASS_AUTO_ROT_DEFAULT 2
+
+# don't share IOMCU DMA
+DMA_NOSHARE UART8* SPI1* TIM*UP*
+
+DMA_PRIORITY SDMMC* UART8* ADC* SPI1* TIM*
+
+# enable FAT filesystem support (needs a microSD defined via SDMMC)
+define HAL_OS_FATFS_IO 1
+
+define HAL_BOARD_LOG_DIRECTORY "/APM/LOGS"
+define HAL_BOARD_TERRAIN_DIRECTORY "/APM/TERRAIN"
+
+ROMFS io_firmware.bin Tools/IO_Firmware/iofirmware_lowpolh.bin
+# as pushed to master, this will actually be a version of the iofirmware built for the F103 @ 8Mhz, but leaving this as iofirmware_lowpolh.bin to match the current build instructions
+
+# uncomment block below to enable a 2nd MS5611 baro on SPI5
```

Comment:
does this 2nd baro exist? if not then just remove this copied block.

Reply:
removed